.github/pull_request_template.md

```diff
@@ -1,23 +1,35 @@
+- [ ] Update **documentation** as appropriate? (e.g [README.md](../blob/main/README.md), etc.)
+- [ ] **Run the app** and try the changes?
+- [ ] Pull in the latest changes from the **main** branch and **squash** your commits before assigning a reviewer[^pullSquash]
+
+> **Note**
+> It is good practice to provide before and after UI  **screenshots** in the description of this PR This is only applicable for changes that modify the UI.
 
 # Reviewer
 
-- [ ] Checklist review: Did you go through the code with the [Code Review Guidelines](../blob/main/docs/CODE_REVIEW_GUIDELINES.md) checklist?
-- [ ] Ad hoc review: Did you perform an ad hoc review?  _In addition to a first pass using the code review guidelines, do a second pass using your best judgement and experience which may identify additional questions or comments. Research shows that code review is most effective when done in multiple passes, where reviewers look for different things through each pass._
-- [ ] Automated tests: Did you review the automated tests?
-- [ ] Manual tests: Did you review the manual tests?
-- [ ] How is code coverage affected? _We encourage you to compare coverage before and after changes and when possible, leaving it in a better place._
-- [ ] Documentation: Did you review Docs, [README.md](../blob/main/README.md), and [Architecture.md](../blob/main/docs/Architecture.md) as appropriate?
-- [ ] Run the app: Did you run the app and try the changes? _While the CI server runs the app to look for build failures or crashes, humans running the app are more likely to notice unexpected log messages, UI inconsistencies, or bad output data. Perform this step last, after verifying the code changes are safe to run locally._
\ No newline at end of file
+- [ ] Check the code with the [Code Review Guidelines](../blob/main/docs/CODE_REVIEW_GUIDELINES.md) **checklist**
+- [ ] Perform an **ad hoc review**[^adHoc]
+- [ ] Review the the **automated tests**
+- [ ] Review the **manual tests**
+- [ ] Review the **documentation**, [**README.md**](../blob/main/README.md), and [**Architecture.md**](../blob/main/docs/Architecture.md) as appropriate
+- [ ] Run the app and try the changes[^runTheApp]
+
+> **Note**
+> We encourage you to compare coverage before and after changes and when possible, leaving it in a better place.
+
+[^selfReview]: _Code often looks different when reviewing the diff in a browser, making it easier to spot potential bugs._
+[^manTest]: _While we aim for automated testing of the application, some aspects require manual testing. If you had to manually test something during development of this pull request, write those steps down._
+[^coverage]:_While we are not looking for perfect coverage, the tool can point out potential cases that have been missed. **Code coverage** can be generated with: `./gradlew check` for Kotlin modules and `./gradlew connectedCheck -PIS_ANDROID_INSTRUMENTATION_TEST_COVERAGE_ENABLED=true` for Android modules._
```

Comment:
We can probably remove the bold around `**Code coverage**` now that this is under a footnote.